addons/binding/org.openhab.binding.meterreader/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="meterreader" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>MeterReader Binding</name>
+	<description>
+	            The meter reader binding is able to read SML messages (PUSH) and supports IEC 62056-21 modes A,B,C (PULL)and D (PUSH).
+    </description>
```

Comment:
Please apply the auto-formatter on your XML files, they all look a bit messed up with regards to tabs+spaces.

Reply:
yes, sure. Didn't see it in the IDE.

Reply:
Done!